src/shortestpaths/dijkstra.jl

```diff
@@ -116,3 +116,45 @@ end
 
 dijkstra_shortest_paths(g::AbstractGraph, src::Integer, distmx::AbstractMatrix = DefaultDistance(); allpaths=false, trackvertices=false) =
 dijkstra_shortest_paths(g, [src;], distmx; allpaths=allpaths, trackvertices=trackvertices)
+
+struct MultipleDijkstraState{T, U<:Integer}<:AbstractPathState
+    dists::Matrix{T}
+    parents::Matrix{U}
+end
+
+function multisource_dijkstra_shortest_paths(
+    g::AbstractGraph,
+    sources::Vector{U} = collect(vertices(g)),
+    distmx::AbstractMatrix{T} = DefaultDistance();
+    parallel=false,
```

Comment:
Let's make this `parallel_multisource_dijkstra_shortest_paths` and get rid of the `parallel` argument for now.